srcpkgs/gnome-flashback/template

```diff
@@ -1,20 +1,28 @@
 # Template file for 'gnome-flashback'
 pkgname=gnome-flashback
-version=3.34.2
+version=3.36.3
 revision=1
 build_helper=gir
 build_style=gnu-configure
-configure_args="--enable-compile-warnings=minimum"
-hostmakedepends="pkg-config automake glib-devel gettext"
-makedepends="gtk+3-devel gsettings-desktop-schemas-devel
- libglib-devel gdk-pixbuf-devel upower-devel ibus-devel polkit-devel
- pulseaudio-devel libcanberra-devel libxcb-devel libX11-devel pango-devel
- libxkbfile-devel gnome-bluetooth-devel gnome-desktop-devel"
+configure_args="--with-compiz-session --enable-compile-warnings=minimum"
+hostmakedepends="pkg-config automake glib-devel gettext libtool"
```

Comment:
Also needs `gettext-devel` for executing `autopoint`.

Reply:
Added in `hostmakedepends`.